api/main.py

```diff
@@ -1111,6 +1111,11 @@ class CI_Measurement(BaseModel):
     cb_company_uuid: Optional[str] = ''
     cb_project_uuid: Optional[str] = ''
     cb_machine_uuid: Optional[str] = ''
+    lat: Optional[str] = ''
```

Comment:
is that really a string and not a float?

Reply:
Yes it is. When coding I thought about it and then decided to use string as floats are not guaranteed to be precise and I thought we are not really going to do anything with the data computational so I decided that it could be a string. A float would be smaller in footprint though. I also thought about using something like postgis as we could then do distance measurements between where the jobs are run. But decided to KISS it.

Reply:
Thanks for the clarification. Makes sense